accessibility_monitoring_platform/apps/audits/forms.py

```diff
@@ -326,7 +326,6 @@ class AuditStatement1UpdateForm(VersionForm):
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         self.fields["add_contact_email"] = AMPCharFieldWide(label="Email")
-        self.fields["add_contact_notes"] = AMPTextField(label="Notes")
```

Comment:
Is it possible to keep this field and the data in the database?

Reply:
I don't know how that would work.

Where would notes entered here be stored? If in `Case.contact_notes` then would existing content be shown here for the user to edit? If not then the users would be blindly entering notes which get appended to existing text.